MutationObserver.js

```diff
@@ -199,6 +199,43 @@ window.MutationObserver = window.MutationObserver || (function(undefined) {
         };
     }
 
+    /**
+     * Checks if browser provides value for style attribute
+     *
+     * @return {boolean}
+     */
+    function checkAttributeStyleReturnsValue() {
+        var div = document.createElement("div");
+        div.style.display = "block";
+        return div.attributes.style.value !== "null";
+    }
+
+    /**
+     * Gets an attribute value
+     *
+     * @param {HTMLElement} el
+     * @param {Attr} attr
+     * @return {String} an attribute value
+     */
+    function getAttributeSimple(el, attr) {
+        return attr.value;
+    }
+
+    /**
+     * Gets an attribute value with special hack for style attribute
+     *
+     * @param {HTMLElement} el
+     * @param {Attr} attr
+     * @return {String} an attribute value
+     */
+    function getAttributeWithStyleHack(el, attr) {
+        if (attr.name !== "style")
+            return attr.value;
+        return el.style.cssText;
```

Comment:
Could we do `el.style[attr]` for the style case? Could be wrong but I think that should work in all browsers